<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsTemporalControllerDockWidgetBase</class>
 <widget class="QgsDockWidget" name="QgsTemporalControllerDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1205</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gsDockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="mSettings">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../images/images.qrc">
          <normaloff>:/images/themes/default/propertyicons/settings.svg</normaloff>:/images/themes/default/propertyicons/settings.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="mCurrentRangeLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="mPreviousButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../images/images.qrc">
          <normaloff>:/images/themes/default/temporal_navigation/previous.svg</normaloff>:/images/themes/default/temporal_navigation/previous.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mBackButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../images/images.qrc">
          <normaloff>:/images/themes/default/temporal_navigation/back.svg</normaloff>:/images/themes/default/temporal_navigation/back.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mStopButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../images/images.qrc">
          <normaloff>:/images/themes/default/temporal_navigation/stop.svg</normaloff>:/images/themes/default/temporal_navigation/stop.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mForwardButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../images/images.qrc">
          <normaloff>:/images/themes/default/temporal_navigation/forward.svg</normaloff>:/images/themes/default/temporal_navigation/forward.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mNextButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../images/images.qrc">
          <normaloff>:/images/themes/default/temporal_navigation/next.svg</normaloff>:/images/themes/default/temporal_navigation/next.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Time steps of</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QgsDoubleSpinBox" name="mSpinBox"/>
      </item>
      <item>
       <widget class="QComboBox" name="mTimeStepsComboBox">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Currently set from </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="mStartDateTime">
        <property name="displayFormat">
         <string>M/d/yyyy h:mm AP</string>
        </property>
        <property name="timeSpec">
         <enum>Qt::UTC</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>to </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="mEndDateTime">
        <property name="displayFormat">
         <string>M/d/yyyy h:mm AP</string>
        </property>
        <property name="timeSpec">
         <enum>Qt::UTC</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mSetToProjectTimeButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../images/images.qrc">
          <normaloff>:/images/themes/default/propertyicons/temporal.svg</normaloff>:/images/themes/default/propertyicons/temporal.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QSlider" name="mSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDockWidget</class>
   <extends>QDockWidget</extends>
   <header>qgsdockwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
